x-pack/legacy/plugins/siem/server/lib/timeline/saved_object.ts

```diff
@@ -271,15 +297,17 @@ export const convertStringToBase64 = (text: string): string => Buffer.from(text)
 // then this interface does not allow types without index signature
 // this is limiting us with our type for now so the easy way was to use any
 
-const timelineWithReduxProperties = (
+export const timelineWithReduxProperties = (
   notes: NoteSavedObject[],
   pinnedEvents: PinnedEventSavedObject[],
   timeline: TimelineSavedObject,
-  userName: string
+  userName?: string
```

Comment:
why userName becomes a conditional attributes?

Reply:
My bad, I was doing some testing and forgot to put it back.